indico/modules/events/registration/templates/emails/registration_creation_to_registrant.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'events/registration/emails/base_registration_details.html' %}
+{% block subject_name %}{% endblock %}
+
+{% block subject_message -%}
+    Registration
+{%- endblock %}
+
+{% block registration_header -%}
+    {% set event = registration.registration_form.event %}
+    <p>Dear {{ registration.full_name }},</p>
+    <p>
+        {% block registration_notification %}
+            Congratulations! Your registration "{{ registration.registration_form.title }}" for the event "{{ event.title }}" has been {% block registration_action %}recorded{% endblock %} successfully.
+            {%- if registration.state.name == 'pending'%}
+                However, it's not complete yet since it needs to be approved by an organizer. You will be notified when that happens.
+            {%- elif registration.state.name == 'unpaid' %}
+                Please, proceed to the checkout <a href="{{ url_for('event_registration.display_regform_summary', registration.locator.registrant, _external=true) }}">here</a> in order to complete it.
```

Comment:
For the record: http://english.stackexchange.com/questions/389/in-what-cases-should-i-use-a-comma-after-please